### PR TITLE
Update members.xml

### PR DESCRIPTION
**Pull request by spykelionel**

### CODE REVIEW

**Comment by elroykanye** [comment]
fetch from upstream and try this again :)

**Comment by spykelionel** [comment]
done

File: members.xml [hunk 1]
```diff
@@ -31,5 +31,11 @@
             <email>mofirojean@mail.com</email>
             <occupation>Student</occupation>
         </member>
+      <member>
+            <id>2</id>
+            <name>NDI Lionel</name>
+            <email>ndilionel6@mail.com</email>
+            <occupation>Student</occupation>
+        </member>
```
Comment (elroykanye) [inline members.xml:39]: Could you adjust the indentation too?

**Review by elroykanye — CHANGES_REQUESTED** [review]
Thanks

File: members.xml (hunk 1 above)
Comment (spykelionel) [reply members.xml:39]: oh sure.

**Review by elroykanye — APPROVED** [review]

**Pull request merged**